RecoTracker/MeasurementDet/plugins/MeasurementTrackerESProducer.cc

```diff
@@ -155,9 +158,11 @@ MeasurementTrackerESProducer::MeasurementTrackerESProducer(const edm::ParameterS
 
   //FIXME:: just temporary solution for phase2!
   auto phase2 = p.getParameter<std::string>("Phase2StripCPE");
+  auto phase2Matcher = p.getParameter<std::string>("Phase2HitMatcher");
   if (not phase2.empty()) {
     usePhase2_ = true;
     c.setConsumes(phase2TrackerCPEToken_, edm::ESInputTag("", phase2));
+    c.setConsumes(phase2matcherToken_, edm::ESInputTag("", phase2Matcher));
```

Comment:
In recent IBs these could be (and eventually should become)
```suggestion
    phase2TrackerCPEToken_ = c.consumes(edm::ESInputTag("", phase2));
    phase2matcherToken_ = c.sconsumes(edm::ESInputTag("", phase2Matcher));
```

Reply:
Done